spaceship/README.md

```diff
@@ -133,6 +133,15 @@ export FASTLANE_SESSION='---\n- !ruby/object:HTTP::Cookie\n  name: DES5c148586df
 
 Copy everything from `---\n` to your CI server and provide it as environment variable named `FASTLANE_SESSION`.
 
+#### Bypass trusted device and use SMS for verification
+
+Apple will not send a SMS code to your phone if you have a trusted device configured for your Apple account when you try to generate a web session with _fastlane_ — instead a code will be displayed on one of your account's trusted devices. This can be problematic if you are trying to authenticate but don't have access to a trusted device. Take the following steps to circumvent the device and use SMS instead:
```

Comment:
I read through this one more time and think ONE more rework will make this 💯 😊

>If you have a trusted device configured, Apple will not send a SMS code to your phone for your Apple account when you try to generate a web session with _fastlane_. Instead, a code will be displayed on one of your account's trusted devices. This can be problematic if you are trying to authenticate but don't have access to a trusted device. Take the following steps to circumvent the device and use SMS instead: